.github/workflows/perf_micro.yml

```diff
@@ -58,7 +65,8 @@ jobs:
       - self-hosted
       - Linux
       - x86_64
-      - ${{ (github.event_name != 'pull_request' ||
+      - ${{ ((github.event_name != 'pull_request' ||
```

Comment:
I am a little confused by this change:

Looks like the condition 
```
(github.event_name != 'pull_request' || github.event_name != 'pull_request_target')
```
is always true (`pull_request_target != pull_request` and `pull_request != pull_request_target`).

I am not sure that we can use `github.event.pull_request.head.repo.full_name`, when the workflow is triggered by the `pull_request_target` event.

I suppose we can drop this change since the check for `pull_request_target` is made above.

Reply:
reverted:

```diff
--- a/.github/workflows/perf_micro.yml
+++ b/.github/workflows/perf_micro.yml
@@ -73,8 +73,7 @@ jobs:
       - self-hosted
       - Linux
       - x86_64
-      - ${{ ((github.event_name != 'pull_request' ||
-              github.event_name != 'pull_request_target') ||
+      - ${{ (github.event_name != 'pull_request' ||
           github.event.pull_request.head.repo.full_name == github.repository) &&
           'performance' || 'regular' }}
 ```